src/main/java/xyz/nucleoid/creator_tools/workspace/editor/ServersideWorkspaceEditor.java

```diff
@@ -72,6 +79,36 @@ public boolean useRegionItem() {
         return true;
     }
 
+    @Override
+    public boolean useRegionVisibilityFilterItem(Collection<String> regions) {
+        Predicate<String> oldFilter = this.filter;
+        this.filter = regions == null || this.player.isSneaking() ? NO_FILTER : regions::contains;
```

Comment:
Hmm, it feels a bit odd for the sneaking check handling to be here- I'd imagine it to be more of a detail of the item used?

Reply:
I placed sneaking handling here because the add region functionality also handles sneaking here, but I can move it out if I refactor the `useRegionVisibilityFilterItem` method into something like `applyFilter`.

Reply:
I moved sneaking handling out of this method in 1fb476ab0232e78158359a7b7fcf9076d4a1c4db.